content/case_histories/mt_isa/data.rst

```diff
@@ -8,6 +8,7 @@ The `MIMDAS`_ system collects both a pole-dipole (P-DP) and a dipole-pole
 collected through the system. The data are viewed prior to quality control
 measures.
 
+.. note :: The data in UBC format can be `downloaded here <https://github.com/geoscixyz-data/Mt-Isa/raw/master/data.zip>`_
```

Comment:
would it make sense just to use the repo-url? That way we can add a readme and such later on and have this be the first thing the reader sees? It is fine either way, up to you!

Reply:
Roger that